apps/api/src/app/bridge/e2e/sync.e2e.ts

```diff
@@ -112,7 +113,6 @@ describe('Bridge Sync - /bridge/sync (POST)', async () => {
 
     expect(workflowData.steps.length).to.equal(1);
     expect(workflowData.steps[0].stepId).to.equal('send-email');
-    expect(workflowData.steps[0].uuid).to.equal('send-email');
```

Comment:
@djabarovgeorge This tells us that the mock data generator for the e2e is probably not realistic and needs an update. Let's fix that in a follow up PR.

Reply:
It is not related to the mock, this param was generated in the API.
it is related to the fact that we do not create this duplicated uuid in the workflow entity anymore, so there is nothing to check.